CHANGELOG.md

```diff
@@ -139,6 +139,7 @@ ___
 - Fix LiveQuery server crash when using $all query operator on a missing object key (Jason Posthuma) [#7421](https://github.com/parse-community/parse-server/pull/7421)
 - Added runtime deprecation warnings (Manuel Trezza) [#7451](https://github.com/parse-community/parse-server/pull/7451)
 - Add ability to pass context of an object via a header, X-Parse-Cloud-Context, for Cloud Code triggers. The header addition allows client SDK's to add context without injecting _context in the body of JSON objects (Corey Baker) [#7437](https://github.com/parse-community/parse-server/pull/7437)
+- Added Cloud trigger `Parse.Cloud.beforeUnsubscribe()` [#7419](https://github.com/parse-community/parse-server/pull/7419)
```

Comment:
Could you change this to

> Added Cloud Trigger for LiveQuery event before subscribe

Reply:
Can you please remove the function name from the changelog and just use:

> Added Cloud Trigger for LiveQuery event before unsubscribe (Samuel Denis-D'Ortun) [#7419](https://github.com/parse-community/parse-server/pull/7419)

or 

> Added Cloud Trigger for LiveQuery event before unsubscribe (sadortun) [#7419](https://github.com/parse-community/parse-server/pull/7419)

whatever you prefer :-)

Reply:
@mtrezza  Updated.